### PR TITLE
Fixing the python and js packaging

### PR DESCRIPTION
**Pull request by mistercrunch**
@williaster 
This makes it such that we don't have to ship the whole `node_modules` with the python package, to do that I had to move a bunch of `css` html references to the `js` require notation. I managed to do it for everything but the `font-awesome` one.

This also cranks up the version to 0.8.0, I think it's ready to ship to Pypi.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,4 +1,8 @@
 recursive-include panoramix/templates *
 recursive-include panoramix/static *
+recursive-exclude panoramix/static/assets/node_modules *
```
Comment (mistercrunch) [inline MANIFEST.in]: This excludes the thousands of files in that folder to ship with the python package

File: panoramix/assets/webpack.config.js [hunk 2]
```diff
@@ -30,9 +30,14 @@ var config = {
         include: APP_DIR,
         loader: "style-loader!css-loader"
       },
-    /* for require('*.less') */
+    /* for css linking images */
+      { test: /\.png$/, loader: "url-loader?limit=100000" },
+      { test: /\.jpg$/, loader: "file-loader" },
+      { test: /\.gif$/, loader: "file-loader" },
+    /* for font-awesome */
       { test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "url-loader?limit=10000&minetype=application/font-woff" },
```
Comment (mistercrunch) [inline panoramix/assets/webpack.config.js]: this still doesn't work...

File: MANIFEST.in (hunk 1 above)
Comment (williaster) [inline MANIFEST.in]: gotcha, and they're already included in the bundled js files so are totally unnecessary

File: panoramix/assets/webpack.config.js (hunk 2 above)
Comment (williaster) [inline panoramix/assets/webpack.config.js]: not sure what you mean by that. does it still compile the `.less` / throw errors during `npm run dev|prod`? this was necessary because it's compiling all of the bootstrap variables, if that works it should be okay.

File: panoramix/templates/panoramix/dashboard.html [hunk 3]
```diff
@@ -66,7 +65,7 @@ <h4 class="modal-title" id="myModalLabel">Dashboard CSS</h4>
       </div>
     </div>
 </div>
-<div class="gridster content_fluid">
+<div class="gridster content_fluid" style="visibility: hidden;">
```
Comment (williaster) [inline panoramix/templates/panoramix/dashboard.html]: css rule instead of inline style! why is this necessary? and is `visibility: hidden;` what you want or `display: none;`?

File: panoramix/assets/webpack.config.js (hunk 2 above)
Comment (mistercrunch) [inline panoramix/assets/webpack.config.js]: oh I thought it was an attempt at getting `font-awesome` to work as the failure is around `woff` files and such...

File: panoramix/templates/panoramix/dashboard.html (hunk 3 above)
Comment (mistercrunch) [inline panoramix/templates/panoramix/dashboard.html]: Well I had the `visibility:hidden` in `panoramix.css` but it would flicker, while the javascript `.gridster` call alters the css or classes the pre and post css changes would show for a fraction of a second. 

When the `visibility:hidden` is in `panoramix.css` it flickers, but not when it's inline... Let's land this so I can get this release out and we can revisit later. Also the `display:none` option might be better but the jquery `$.show()` would animate while there's already a lot going on so I went for this approach instead. 

I think the location of the js (head vs tail) might have something to do with it as well, maybe all of our `entry` scripts should be head since they contain css now. Anyhow, I'm merging and pushing a release as this works well and we can work out the details later.

**Review by Laxonj — COMMENTED** [review]
Ok
